blackbox/docs/sql/ddl.txt

```diff
@@ -512,15 +512,38 @@ definition:
 Disable indexing
 ................
 
-Indexing can be turned off by using the ``INDEX OFF`` column
-definition. Without an index the column can never be hit by a query,
-and is only available as a result column::
+Indexing can be turned off by using the ``INDEX OFF`` column definition.
+Consider that a column without an index can only be used as a result column,
+but it cannot be queried.
```

Comment:
i think it is now wrong. since it can be queried, since it is allowed to issue a query. however it will never produce a hit